tools/docker/Dockerfile

```diff
@@ -56,7 +56,7 @@ RUN git clone https://github.com/kiyo-masui/bitshuffle.git bitshuffle && \
 RUN apt-get update && \
     apt-get -y install libopenblas-dev liblapack-dev liblapacke-dev && \
     apt-get clean && apt-get autoclean && \
-    git clone https://bitbucket.org/blaze-lib/blaze.git blaze
+    git clone https://bitbucket.org/blaze-lib/blaze.git blaze && cd blaze && git pull && cd ..
```

Comment:
We should add this to the MacOS build as well just in case it's also caching things.